packages/gestalt/src/Sticky.js

```diff
@@ -20,19 +21,28 @@ type Threshold =
 
 type Props = {|
   children: React.Node,
+  zIndex?: Indexable,
   dangerouslySetZIndex?: {| __zIndex: number |},
   ...Threshold,
 |};
 
+const DEFAULT_ZINDEX = new FixedZIndex(1);
```

Comment:
This will result in every sticky now having a default zIndex of 1 but before some of them may not have had a zIndex. Is this required?